### PR TITLE
Bug 1430417 Added information on ipsec performance considerations

### PR DESCRIPTION
**Pull request by bfallonf**
As per: https://bugzilla.redhat.com/show_bug.cgi?id=1430417#c4

cc @jeremyeder @dcbw  @mburke5678 

I'm not sure what else should be added here? There's not much information, and not really anything particularly new for the reader. Should there be more? Jeremy, is this going to be added to in the [trello work](https://trello.com/c/xt2Y6Epu/415-networking-quantify-perf-impact-of-ipsec-8) you mention?

### CODE REVIEW

**Comment by bfallonf** [comment]
Bump @jeremyeder @dcbw @mburke5678

File: scaling_performance/network_optimization.adoc [hunk 1]
```diff
@@ -137,4 +137,20 @@ osm_cluster_network_cidr=10.128.0.0/10
...
 
-Please refer to the supportability limits in the {product-title} documentation for node/pod limits for the version of software you are installing.
+See the supportability limits in the {product-title} documentation for node/pod limits for the version of software you are installing.
+
+[[scaling-performance-optimizing-ipsec]]
+== Optimizing IPSec
+
+Because encrypting and decrypting node hosts uses CPU power, performance will be
+affected both in throughput and CPU usage on the nodes when encryption is
+enabled, no matter what IP security system is used.
```
Comment (mburke5678) [inline scaling_performance/network_optimization.adoc]: s/no matter what IP security system is used/regardless of the IP security system being used
"no matter what..." seems a bit slangy.

File: scaling_performance/network_optimization.adoc [hunk 2]
```diff
@@ -137,4 +137,20 @@ osm_cluster_network_cidr=10.128.0.0/10
...
+
+IPSec encrypts traffic at the IP level, before it hits the NIC, protecting
+fields that would otherwise be used for NIC offloading. This means that some NIC
+acceleration features may not be usable when IPSec is enabled and throughput and
+CPU usage will increase to some degree.
+
```
Comment (mburke5678) [inline scaling_performance/network_optimization.adoc]: "to some degree" seems too vague. How about "...CPU usage will increase."

**Comment by mburke5678** [comment]
@bfallonf Couple of quick thoughts on style. Otherwise LGTM.

**Comment by bfallonf** [comment]
Bump @jeremyeder @dcbw

**Comment by jeremyeder** [comment]
LGTM.  I've asked the team to juggle some priorities around to get test data in this config so we can put some numbers to it eventually.

**Comment by bfallonf** [comment]
Thanks @jeremyeder . I'll move forward with this. Let me know if there's anything I can help with regarding the numbers.

@mburke5678 , I put in your suggestions. Things haven't changed since, so I'll count that as the peer review. Merging.

**Comment by bfallonf** [comment]
[rev_history]
|xref:../scaling_performance/network_optimization.adoc#scaling-performance-routing-network-optimization[Routing and Network Optimization]
|Added the xref:../scaling_performance/network_optimization.adoc#scaling-performance-optimizing-ipsec[Optimizing IPSec] section.
%

**Pull request merged**